website/source/assets/javascripts/application.js

```diff
@@ -3,3 +3,6 @@
 
 //= require hashicorp/mega-nav
 //= require hashicorp/sidebar
+//= require hashicorp/analytics
+
+//= require analytics
```

Comment:
Is this different than `hashicorp/analytics`?

Reply:
Yep - `hashicorp/analytics` is a global import from `middleman-hashicorp` which provides utility functions for tracking events across multiple elements. `analytics` is the local script file used just for this site 😁

Reply:
👍 :shipit: